utils/swift_build_support/tests/products/test_llvm.py

```diff
@@ -2,7 +2,7 @@
 #
 # This source file is part of the LLVM.org open source project
 #
-# Copyright (c) 2014 - 2018 Apple Inc. and the LLVM project authors
+# Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
```

Comment:
If this file is part of the LLVM project, then the original header is quite correct.